packages/kernel-utils/CHANGELOG.md

```diff
@@ -7,11 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0]
+
+### Changed
+
+- Bump endo dependencies ([#543](https://github.com/MetaMask/ocap-kernel/pull/543))
+- Bump vite and vitest dependencies ([#516](https://github.com/MetaMask/ocap-kernel/pull/516))
```

Comment:
This does not seem to have affected this package, and dev dependencies are in any event irrelevant for the changelog.